packages/wallets/src/sdk.ts

```diff
@@ -7,20 +7,25 @@ import type { WalletOptions } from "./utils/options.js";
 
 type WalletType = keyof WalletTypeToArgs;
 
+export type CrossmintWalletsOptions = {
+    appId?: string;
+};
+
```

Comment:
Alternatively, we can extend the `Crossmint` class globally. Not sure that would be the best move, as we only really need it in the React Native SDK.